app/controllers/buildings_controller.rb

```diff
@@ -8,4 +8,17 @@ def show
     @building = Building.find(params[:id])
     render 'show'
   end
+
+  ##
+  # Given a building, and an email, create a new building_operator
+  # and mark the given question in the building as delegated, and return a new
+  # building instance.
+  #
+  def delegate_question(building, email)
+    building_operator = BuildingOperator.where(email: email).first_or_create
+    building.answers.where{ |a| a.predelegated? }.each do |answer|
+      answer = Answer.create!(building: building, question: answer.question, building_operator: building_operator)
```

Comment:
so i don't think building_operator exists as a a column anymore.. Pull the latest changes from master and migrate. Arpan changed the association from answer to building_operator/asset_manager into a polymorphic association; I believe the columns are now `user_type` and `user_id`

Reply:
- [ ]